### PR TITLE
Add repeatability buttons and vertical tabs support

### PR DESCRIPTION
**Pull request by suryansh29**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: src/components/accordion/_accordion.scss [hunk 1]
```diff
@@ -61,4 +63,28 @@
   &__panel {
     @include tab-panel;
   }
+
+  &__repeatable-buttons{
```
Comment (amit-mnnit) [inline src/components/accordion/_accordion.scss:67]: Create mixin for this

File: src/components/verticaltabs/_verticaltabs.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
+.cmp-verticaltabs{
+    font-size: $font-m;
+    margin: $field-margin;
+    position: relative;
+  
+  &__tabs-container{
+    display: flex;
+    margin-top: 16px;
+  }
+  &__tabpanel{
```
Comment (amit-mnnit) [inline src/components/verticaltabs/_verticaltabs.scss:10]: please use panel-label mixin here

File: src/components/verticaltabs/_verticaltabs.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    flex: 1;
+    padding: 16px 56px;
+  }
+  &__tabpanel--active{
```
Comment (amit-mnnit) [inline src/components/verticaltabs/_verticaltabs.scss:14]: remove empty class

File: src/components/verticaltabs/_verticaltabs.scss [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    padding: 12px 4px 12px 12px;
+    margin-top: 8px;
+  }
+  &__tab:hover{
+    border-left: 2px solid $light-gray;
+  }
+  &__tab--active {
+    border-left: 2px solid;
+  }
+  &__longdescription {
```
Comment (amit-mnnit) [inline src/components/verticaltabs/_verticaltabs.scss:33]: use description mixin for description, short description and long description

File: src/components/verticaltabs/_verticaltabs.scss [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    margin-top: 16px;
+  }
+  &__tabpanel{
+    flex: 1;
+    padding: 16px 56px;
+  }
+  &__tabpanel--active{
+    
+  }
+  &__tablist{
```
Comment (amit-mnnit) [inline src/components/verticaltabs/_verticaltabs.scss:17]: Create mixin for tablist and tab

File: src/site/_variables.scss [hunk 6]
```diff
@@ -22,6 +22,8 @@ $light-error: rgba(235, 60, 54, 0.05);
 $primary: $dark-gray;
 $dark-primary: $dark-gray;
 $disabled-color: $background-color;
+$bg-gray: #F1F1F1;
```
Comment (amit-mnnit) [inline src/site/_variables.scss:25]: Don’t create new variables

**Review by amit-mnnit — CHANGES_REQUESTED** [review]
Don’t create new variables.

File: src/components/verticaltabs/_verticaltabs.scss (hunk 2 above)
Comment (suryansh29) [reply src/components/verticaltabs/_verticaltabs.scss:10]: done

File: src/components/verticaltabs/_verticaltabs.scss (hunk 3 above)
Comment (suryansh29) [reply src/components/verticaltabs/_verticaltabs.scss:14]: done

File: src/components/verticaltabs/_verticaltabs.scss (hunk 4 above)
Comment (suryansh29) [reply src/components/verticaltabs/_verticaltabs.scss]: done

File: src/site/_variables.scss (hunk 6 above)
Comment (suryansh29) [reply src/site/_variables.scss:25]: done

File: src/components/accordion/_accordion.scss (hunk 1 above)
Comment (suryansh29) [reply src/components/accordion/_accordion.scss]: done

File: src/components/verticaltabs/_verticaltabs.scss (hunk 5 above)
Comment (suryansh29) [reply src/components/verticaltabs/_verticaltabs.scss:14]: Do we need this mixin it is used here only and will not be used for other components?

**Review by amit-mnnit — APPROVED** [review]

**Pull request merged**